modules/Bio/EnsEMBL/Compara/RunnableDB/PairAligner/ParsePairAlignerConf.pm

```diff
@@ -517,16 +517,11 @@ sub get_chunking {
        $dna_collection->{'include_non_reference'} = $default_chunk->{'include_non_reference'};
    }
 
-   #set masking_option if neither masking_option_file or masking_options has been set
-   unless (defined $dna_collection->{'masking_options'} || defined $dna_collection->{'masking_options_file'}) {
-       $dna_collection->{'masking_options'} = $default_chunk->{'masking_options'};
+   #set masking_option if maskinghas not been set
```

Comment:
```suggestion
   #set masking_option if masking has not been set
```

Reply:
We no longer have `'masking_option'` (neither with an 's' at the end) defined:
```suggestion
   #set the masking if it has not been set
```